n_t_with_s/test_snakemake.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
also confused about this file-- can it be removed from the PR?

Reply:
yes